airflow/dags/fda_unii/dag.py

```diff
@@ -0,0 +1,64 @@
+import pendulum
+
+from airflow_operator import create_dag
+from airflow.providers.postgres.operators.postgres import PostgresOperator
+from airflow.decorators import task
+
+from common_dag_tasks import  extract, transform, generate_sql_list, get_ds_folder
+from sagerx import read_sql_file
+
+dag_id = "fda_unii"
+
+dag = create_dag(
+    dag_id=dag_id,
+    schedule="0 4 * * *",
+    start_date=pendulum.yesterday(),
+    catchup=False,
+    concurrency=2,
+)
+
+with dag:
+    url= "https://precision.fda.gov/uniisearch/archive/latest/UNII_Data.zip"
+    ds_folder = get_ds_folder(dag_id)
+
+    extract_task = extract(dag_id,url)
+    transform_task = transform(dag_id)
+
+    @task
+    def get_file_name(data_path) -> str:
```

Comment:
@lprzychodzien - I made this task because the data file inside of the zip file changes every time they update the data.  It has the month and year in the file name. Would this make more sense in common_dag_tasks?

Reply:
You can use the get_files_in_data_folder function already there to pull all of the downloaded data files. It returns a list so you can iterate through it here in the DAG to find the match